agenta-web/src/lib/helpers/formatters.ts

```diff
@@ -5,17 +5,25 @@ const intlNumber = new Intl.NumberFormat("en-US", {
 const intlCurrency = new Intl.NumberFormat("en-US", {
     style: "currency",
     currency: "USD",
-    maximumFractionDigits: 4,
+    maximumFractionDigits: 6,
 })
 
 export const formatNumber = (value = 0) => {
     return intlNumber.format(value)
 }
 
-export const formatCurrency = (value = 0) => {
+export const formatCurrency = (value: number) => {
```

Comment:
I think there is still a use case that we don't cover where the currency is null (and not 0) then indeed we need to display "-"